src/lib/services/store/store.ts

```diff
@@ -0,0 +1,29 @@
+import type { Writable } from 'svelte/store';
+import { writable, get } from 'svelte/store'
+
+const storage = <T>(key: string, initValue: T): Writable<T> => {
```

Comment:
If you move the Store type over here, you can delete that`consts` [file](https://github.com/Rapidmidiex/rmx-web-client/pull/16/files#r1103879170)
```suggestion
type Store =  'CHAT_STORE' |  'THEME_STORE'

const storage = <T>(key: Store, initValue: T): Writable<T> => {
```